src/txn/txn.c

```diff
@@ -572,11 +572,14 @@ __wt_txn_update_oldest(WT_SESSION_IMPL *session, uint32_t flags)
 
         /* Output a verbose message about long-running transactions,
```

Comment:
Can you update the format of this comment? It is currently out of style and should be formatted across the lines correctly.